projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 ################################################################################
 #      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
```

Comment:
also adding the new copyright line which directing to someone who has not done this looks fishy